src/ansys/aedt/core/workflows/templates/pyaedt_utils.py

```diff
@@ -65,7 +65,7 @@ def check_file(file_path, oDesktop):
 
 
 def get_linux_terminal():
-    for terminal in ["x-terminal-emulator", "konsole", "xterm", "gnome-terminal", "lxterminal", "mlterm"]:
+    for terminal in ["x-terminal-emulator", "xterm", "gnome-terminal", "lxterminal", "mlterm"]:
```

Comment:
Why are we dropping KDE terminal ? Is it related to the OS compatible with AEDT ?

Reply:
@SMoraisAnsys There were some env. variables conflicts, and indeed OS compatibility issues